proto/account/service.proto

```diff
@@ -265,7 +265,10 @@ message ListAPIKeysResponse {
 }
 
 message GetAPIKeyBySearchingAllEnvironmentsRequest {
-  string id = 1;
+  reserved 1;
```

Comment:
Since we are using the canary release, if we don't support both cases when migrating to `api_key,` the requests will fail until the canary release is over.

Ideally, it would be to support both cases, so it won't fail.

Reply:
Got it, I have implemented a small logic to support both fields